TestFramework/ExtensionMethods.cs

```diff
@@ -244,6 +244,11 @@ public static IWebElement MoveToElement(IWebDriver driver, By locator, ScenarioC
             }
         }
 
+        public static bool VerifyPageUrl(IWebDriver webdriver, string title)
+        {
+            return webdriver.Url.Contains(title);
```

Comment:
I think we don't need a new method for one line statements

Reply:
Removed it.